pyproject.toml

```diff
@@ -73,6 +73,8 @@ test = [
     "pytest-cov",
     "pytest-sugar",
     "pytest-xdist",
+    "pytest-asyncio",
```

Comment:
you've installed these in `[test]`, but we actually install `[develop]` so they need to be duplicated there